client/lib/domains/email-forwarding/index.js

```diff
@@ -1,14 +1,23 @@
 import emailValidator from 'email-validator';
 import { mapValues } from 'lodash';
+import { hasDuplicatedEmailForwards } from 'calypso/lib/domains/email-forwarding/has-duplicated-email-forwards';
 
-function validateAllFields( fieldValues ) {
+function validateAllFields( fieldValues, existingForwards = [] ) {
 	return mapValues( fieldValues, ( value, fieldName ) => {
```

Comment:
Not to future us: it would be good to refactor this code away from `lodash` when we get a chance.

That's not something we should do in this PR, though!